mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule Bonny.MixProject do
       app: :bonny,
       description: description(),
       version: @version,
-      elixir: "~> 1.9",
```

Comment:
We actually want to support older versions than 1.13, too. This will lead to a warning when compiled for older versions of Elixir.

Reply:
Ah yes, I left a note to myself to check that and forgot.  Do you know how  give it a range?  I tried  `>= 1.7`

Reply:
why change it? Currently we have `elixir: "~> 1.10"`.